digdag-standards/src/main/java/io/digdag/standards/operator/ForEachOperatorFactory.java

```diff
@@ -27,10 +27,13 @@
         implements OperatorFactory
 {
     private static Logger logger = LoggerFactory.getLogger(ForEachOperatorFactory.class);
+    private Limits limits;
 
     @Inject
-    public ForEachOperatorFactory()
-    { }
+    public ForEachOperatorFactory(Limits limits)
```

Comment:
Operators need to get limits from the server through TaskRequest (queue) or through TaskCallbackApi (REST API).